lxd/devices.go

```diff
@@ -431,7 +431,7 @@ func deviceTaskSchedulerTrigger(srcType string, srcName string, srcStatus string
 	}()
 }
 
-func deviceIsDevice(path string) bool {
+func fileIsBlockdev(path string) bool {
```

Comment:
Sure.  I started with that and it sounded funny to me, but your
reasoning makes sense.